pkg/agent/openflow/client.go

```diff
@@ -199,7 +199,7 @@ func (c *client) InstallTunnelFlows(tunnelOFPort uint32) error {
 
 func (c *client) Initialize() error {
 	// Initiate connections to target OFswitch, and create tables on the switch.
-	if err := c.bridge.Connect(maxRetryForOFSwitch); err != nil {
+	if err := c.bridge.Connect(maxRetryForOFSwitch, make(chan struct{})); err != nil {
```

Comment:
Not sure why we make a new channel here. If we don't use this channel, how about creating it inside the `Connect`?

Reply:
The channel should be left to agent to receive to connected or re-connected event. It is used to reset the round id in cookie after introduce reconcilation

Reply:
Good to know, thanks.